### ISSUE TITLE
moving /testing one stepper. 

### ISSUE THREAD

**Issue by rewolff** [KevinOConnor/klipper#3394]
My Delta 3D printer refuses to move until I hit "home". Understandable. 

There is an issue with one of the steppers. I don't know what yet. When I hit "home" it doesn't move. To debug this I'd like to be able to say: 

DEBUG_MOVE_TOWER A -12.34

When things are not moving, things are awkward while homing. The head moves outside the envelope (when it starts low) and when I swap connectors to see if it is the motor or the driver things hit endstops hard and don't stop when they are supposed to if I test by hitting "home"... So hitting home all the time is not a good idea. I much prefer to do something more subtle....

**Comment by klipper-gitissuebot** [comment]
Hi @rewolff,

It did not look like there was a Klipper log file attached to this ticket. The log file has been engineered to answer common questions the Klipper developers have about the software and its environment (software version, hardware type, configuration, event timing, and hundreds of other questions).

Unfortunately, too many people have opened tickets without providing the log. That consumes developer time; time that would be better spent enhancing the software. If this ticket references an event that has occurred while running the software then the Klipper log must be attached to this ticket. Otherwise, this ticket will be automatically closed in a few days.

For information on obtaining the Klipper log file see: https://github.com/KevinOConnor/klipper/blob/master/docs/Contact.md

The log can still be attached to this ticket - just add a comment and attach the log to that comment.

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being.

**Comment by rewolff** [comment]
Totally irrelevant but the bot closes issues that don't have a log. 
[klippy.log](https://github.com/KevinOConnor/klipper/files/5321980/klippy.log)

**Comment by KevinOConnor** [comment]
I use the STEPPER_BUZZ command - see: https://www.klipper3d.org/Config_checks.html .  There are also other options described in the FAQ: https://www.klipper3d.org/FAQ.html#why-cant-i-move-the-stepper-before-homing-the-printer

-Kevin

**Comment by rewolff** [comment]
Aaargh. Found it myself already. Didn't use the right keywords when searching. I have enabled force_move and so far I've saved myself some work: Turns out the towers are incorrectly tagged, so the cable that "looks OK but apparently doesnt' work" was of a tower that did work. (To verify something: i'd swapped the one that I thought didn't work and the one that actually didn't work and got the expected effects.... aargh!)
